src/pages/IndexPage.vue

```diff
@@ -1,15 +1,137 @@
 <template>
```

Comment:
Add simple comments for better readability/orientation inside template code, please.

Reply:
Added comments